contracts/test-utils/src/web3_wrapper.ts

```diff
@@ -31,26 +28,9 @@ export const providerConfigs: Web3Config = {
 export const provider: Web3ProviderEngine = web3Factory.getRpcProvider(providerConfigs);
 provider.stop();
 const isCoverageEnabled = env.parseBoolean(EnvVars.SolidityCoverage);
-const isProfilerEnabled = env.parseBoolean(EnvVars.SolidityProfiler);
```

Comment:
profiler provider is gone now -- only showed up once in any post-V3 code and suspect it was a copy paste since the line had not been touched in 4 years